composer.json

```diff
@@ -7,14 +7,14 @@
         "symfony/debug": "^2.6|^3.0",
         "symfony/process": "^2.6|^3.0",
         "react/event-loop": "^0.4",
-        "react/stream": "^0.4",
-        "react/http": "dev-master#cd15204bd15d106d7832c680e4fb0ca0ce2f5e30",
-        "react/socket-client": "^0.5.0",
-        "react/socket": "^0.5.0",
-        "mkraemer/react-pcntl": "2.0.*",
+        "react/stream": "^0.7.1",
+        "react/http": "dev-master",
```

Comment:
I would suggest targeting the specific commit hash until the next release is tagged :+1:

Reply:
Doesn‘t matter. I‘ll just keep this PR updated until tagged release is available.